.azure-pipelines/all.yml

```diff
@@ -21,18 +21,18 @@ resources:
   - container: dd_agent
     image: gcr.io/datadoghq/agent:latest
     ports:
-      - 8127:8126
+    - 8127:8126
```

Comment:
Can you revert the changes in this file? They are not needed.

Reply:
Can you remove the other changes in this file as well? They are only needed for integrations that have code in the repo.

Reply:
I removed everything. (: